### PR TITLE
new parse_fraction function for simple fractions added with test data

### LINKED ISSUE #59: Support for simple fractions
We can add support for the simple fractions as well like three-fourth, half, etc.
Example:

> parse("I have a three-fourth cup of glass.")
I have a 3/4 cup of glass.

> parse_number("half")
1/2

### PR DESCRIPTION
**Pull request by Manish-210**
See issue #59 
parse_fraction function handles basic fraction.
Some test data has also been added.

Resolves #59

### CODE REVIEW

**Review by Gallaecio — COMMENTED** [review]
What about a mention in the README?

**Comment by Manish-210** [comment]
Thanks, I have included a few examples in the README

**Review by Gallaecio — COMMENTED** [review]
At the moment we have an introductory paragraph for the Usage section that mentions 3 interfaces, followed by 4 numbered interface sections.

I think it may be best to remove that paragraph, and rename those sections to something like:

- Converting numbers in-place
- Parsing a number
- Parsing an ordinal
- Parsing a fraction

**Comment by Manish-210** [comment]
Renaming the sections is good, also I have changed the introductory paragraph a little in the usage section. Let me know if it's ok or not.

File: README.rst
Comment (Gallaecio) [inline README.rst:27]: I think only the text should change, the formatting should remain the same, in line with the rest of the file:

```suggestion
Converting numbers in-place
---------------------------
```

The same applies to the other headers.

Comment (Manish-210) [reply README.rst]: Sorry for that, I have changed the formatting so that it will be in line with the rest of the file.

File: README.rst
Comment (Gallaecio) [inline README.rst:40]:
```suggestion
Parsing a number
----------------
```

Looks slightly better, and otherwise Sphinx logs a warning, I believe.

Comment (Manish-210) [reply README.rst]: Thanks for pointing it out, I have corrected it.:smile:

File: README.rst
Comment (Gallaecio) [inline README.rst:63]:
```suggestion
Converting a fractional number written in words to its corresponding integral fraction. (Support for English only)
```

**Review by Gallaecio — COMMENTED** [review]
Some minor wording issues, and I think this is ready to merge.

File: README.rst
Comment (Manish-210) [reply README.rst]: Thanks, I have corrected these mistakes.

**Review by Gallaecio — APPROVED** [review]
Looks great to me!

**Comment by Manish-210** [comment]
Will there be any other checks before merging this pull request?

**Comment by Gallaecio** [comment]
Let’s wait for @noviluni to find some time to review this as well. If he finds the changes OK as well, we’ll merge.

File: number_parser/parser.py
Comment (noviluni) [inline number_parser/parser.py:269]: Could you add an extra newline here to follow PEP8? :)

File: number_parser/parser.py
Comment (noviluni) [inline number_parser/parser.py:292]:
```suggestion
        if number_before_separator is None or number_after_separator is None:
```

File: number_parser/parser.py
Comment (noviluni) [inline number_parser/parser.py:295]:
```suggestion
        return f'{number_before_separator}/{number_after_separator}'
```

File: number_parser/parser.py
Comment (noviluni) [inline number_parser/parser.py:278]:
```suggestion
    FRACTION_SEPARATORS = ["divided by", "over", "by", "/"]
```

I prefer to use capital letters here. Ideally, in the future this will come from the lang_data (`en.json`), we can work on this on a new, separated, PR.

**Review by noviluni — COMMENTED** [review]
@Manish-210 Good job! :rocket: 

Sorry for my late review. I really like the approach you used and I think it will be really useful. Thanks for updating the README too.

I added some notes, let me know once you address them and I will proceed with merging this :smile:

File: number_parser/parser.py
Comment (Manish-210) [reply number_parser/parser.py:278]: Sure, It also has to incorporate the fraction separators in other languages

**Comment by Manish-210** [comment]
Thanks, @noviluni, let me know any other changes.

File: number_parser/parser.py
Comment (Gallaecio) [inline number_parser/parser.py:268]: This is not what @noviluni meant, he meant after the last return. PEP8 asks that top-level function definitions are separated by 2 empty lines.

Comment (Manish-210) [reply number_parser/parser.py]: Sorry for that misinterpretation, I have corrected it.

**Review by noviluni — APPROVED** [review]

**Comment by noviluni** [comment]
Thanks @Manish-210! :smile:

**Pull request merged**
